### ISSUE TITLE
Copy protection?

### ISSUE THREAD

**Issue by ar2013ps** [koying/SPMC#368]
HI, just a little question:
Trying to play a bluray rip of Allegiant (The movie), it stops after 10 min of playback, does anyone if it has anything todo with the copy protection, and what can be done in that case
thanks

**Comment by heymoe** [comment]
Yeah, this is sooooooo not a question for the SPMC bug / issue report forum.
